### DISCUSSION TITLE
Cannot read Hive Data Dome after installing 2.6

### DISCUSSION THREAD

**Discussion by altorico** [artisan-roaster-scope/artisan#853] [General]
Hi, just upgraded to 2.6, fired it up and now I can't read my Hive roaster data dome. Had no problems with 2.4.6. 
Is there a known issue?
Running Ubuntu Linux 20.04.

**Comment by bigl00z3** [comment]
Looking at the website the thermocouple reader is not provided with the roaster. 
What are you using to read your Hive thermocouple? Phidgets? An other reader?

Make sure that you have the latest driver for your reader especially if they are [Phidgets](https://www.phidgets.com/docs/OS_-_Linux).

Nico

**Reply by altorico** [reply]
Thanks for the tips. The Hive Data Dome I believe comes with the reader.
With the 2.4.6 Artisan used Phidget 1051 interface to read the Hive and had no issues with it.
Will check with the maker.
Cheers

**Reply by bigl00z3** [reply]
They were quite a few changes both on the Phidget and Artisan side since last release. 
Updating the [Phidget libraries](https://www.phidgets.com/docs/OS_-_Linux) to the latest version may potentially fix your issue.

I am running on a Raspberry pi, and I had some issues with my Phidget when on 2.4.7. After updating the libraries and using the latest Artisan version all is working very well again.


[https://www.phidgets.com/downloads/phidget22/libraries/linux/libphidget22.tar.gz](https://www.phidgets.com/downloads/phidget22/libraries/linux/libphidget22.tar.gz)
Unpack those and then install via: ./configure;sudo make;make install

Nico

**Reply by altorico** [reply]
Thanks Nico, I downloaded the libs last night.  Will try later and report.

**Reply by altorico** [reply]
Ha! Well, you were right, my friend. Updating the phidget libs did the trick! I am up on running with 2.6. Enjoying the updates!
Thanks for all the support.

Best regards.

Ghena
